libqtile/widget/__init__.py

```diff
@@ -37,66 +39,85 @@
 from libqtile.widget.textbox import TextBox  # noqa: F401
 from libqtile.widget.windowname import WindowName  # noqa: F401
 
+LAZY_IMPORT_SPEC = {
```

Comment:
You could setup a
```
LAZY_MODULE_SPEC = {
    widget: module for module, widgets in LAZY_IMPORT_SPEC.items() for widget in widgets
}
```
that would make the look-ups performed in the `__getattr__` a lot easier to resolve.

Reply:
https://github.com/qtile/qtile/pull/1675#issuecomment-621884260